tests/acceptance/wcag/utils.js

```diff
@@ -0,0 +1,44 @@
+
+/**
+ * Waits until the content of the page stops changing.
```

Comment:
For my own edification, the `goto` method does not wait until the DOM of the page is fully loaded? Instead of rolling our own wait here, could we use `page.waitForNavigation()`? It's referenced a couple places:
- https://stackoverflow.com/questions/52497252/puppeteer-wait-until-page-is-completely-loaded
- https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagewaitfornavigationoptions

Reply:
I think we tried that before in theme and it didn't work well so we resolve it with our custom wait. I tested it out locally just now with different network and DOM content loaded options, it's always stuck on the first page.